src/cbl-mariner/2.0/cross/arm64/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-crossdeps-arm-local
+ARG ROOTFS_DIR=/crossrootfs/arm64
+
+RUN tdnf install -y debootstrap
+
+RUN /scripts/eng/common/cross/build-rootfs.sh arm64 xenial --skipunmount
+
+# Build compiler-rt profile library for PGO instrumentation
+COPY --from=mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-crossdeps-artifacts-local \
```

Comment:
You can't reference a local tag in the `--from` option. This only works if the tag exists in a registry. Instead, you'll need to define a stage from this image name. Something like this:

```Dockerfile
FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-crossdeps-artifacts-local AS artifacts


FROM mcr.microsoft.com/dotnet-buildtools/prereqs:cbl-mariner-2.0-crossdeps-arm-local
ARG ROOTFS_DIR=/crossrootfs/arm64

RUN tdnf install -y debootstrap

RUN /scripts/eng/common/cross/build-rootfs.sh arm64 xenial --skipunmount

# Build compiler-rt profile library for PGO instrumentation
COPY --from=artifacts /llvm-project.src.tar.xz .
```

I'm not sure our infrastructure can support a Dockerfile like this. Give it a shot though and see.